src/components/SkillPage/SkillPage.js

```diff
@@ -0,0 +1,17 @@
+import './SkillPage.css';
+
+function SkillPage({skill}) {
+  const {index, name, desc, ability_score, url} = skill
```

Comment:
Okay love to see this! then you don't have to do `skill.index, skill.name` etc...